src/fmeval/transforms/summarization_accuracy_metrics.py

```diff
@@ -247,23 +277,31 @@ class BertScore(SummarizationAccuracyMetric):
 
     def __init__(
         self,
-        target_output_keys: List[str],
-        model_output_keys: List[str],
         output_keys: List[str],
+        model_output_keys: List[str],
         allow_duplicate_input_keys: bool,
-        bertscore_model: Union[BertscoreHelperModel, ActorHandle],
+        target_output_keys: Optional[List[str]] = None,
+        target_output_keys_provider: str = "",
+        bertscore_model: Union[BertscoreHelperModel, ActorHandle] = BertscoreHelperModel(BERTSCORE_DEFAULT_MODEL),
```

Comment:
We need to put `target_output_keys_provider` as the last argument, or else existing calls to this method will break.

Reply:
Discussed with @kirupang-code offline. We need to use this ordering (which is backwards-incompatible) in order for deserialization of these object instances to work properly (since we rely on a specific ordering of the arguments to `__init__`). The existing calls to `BertScore.__init__` in our own code have been updated as necessary, but users who interact with `BertScore` objects directly will be impacted.

We should keep track of this before the next release.